frontend/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
We should probably delete most of this and stick up a boiler plate "Hello this is the frontend, read the ../README.md" please.